ReactNativeClient/lib/services/KeymapService.ts

```diff
@@ -0,0 +1,202 @@
+		{ accelerator: 'CommandOrControl+A', command: 'selectAll', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+B', command: 'textBold', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+I', command: 'textItalic', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+K', command: 'textLink', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+`', command: 'textCode', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Shift+T', command: 'insertDateTime', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: shim.isMac() ? 'Shift+Command+F' : 'F6', command: 'focusSearch', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+F', command: 'showLocalSearch', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Shift+S', command: 'focusSidebar', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Shift+L', command: 'focusNoteList', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Shift+N', command: 'focusNoteTitle', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Shift+B', command: 'focusNoteBody', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: shim.isMac() ? 'Option+Command+S' : 'F10', command: 'toggleSidebar', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+L', command: 'toggleVisiblePanes', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+0', command: 'zoomActualSize', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+E', command: 'startExternalEditing', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Alt+T', command: 'setTags', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+,', command: 'options', platforms: ['linux', 'win32', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+G', command: 'goToAnything', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'F1', command: 'openWebsite', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		// Keymap items with null "accelerator" property allows users to set their own Accelerators
+		// i.e. { accelerator: null, command: 'someCommand', platforms: ['darwin', 'win32', 'linux', 'freebsd'] }
+	];
+
+	constructor() {
+		super();
+
+		this.keymap = KeymapService.initKeymap();
+		this.logger_ = new Logger();
+	}
+
+	getAccelerator(command: string) {
+		const item = this.keymap[command];
+		// It's okay to provide a null Accelerator for Electron menu items by default
+		return (item == null) ? null : item.accelerator;
```

Comment:
Please only use strict equality `===`. And I feel there should be not any missing accelerator - if you try to get a command and it's not in the list that should be an error.